### PR TITLE
Get currentAssignment explicitly instead of from first object of assignments

### PR DESCRIPTION
**Pull request by mjoyce91**
Resolves TM-2011

### CODE REVIEW

File: talentmap_api/fsbid/services/client.py [hunk 1]
```diff
@@ -205,10 +208,6 @@ def fsbid_clients_to_talentmap_clients(data):
         if current_assignment.get('position', None) is not None:
             position = current_assignment.get('position', None)
 
-    if current_assignment is not None and assignments and type(assignments) is type([]):
-        # remove the current assignment, since we'll pass it with current_assignment
-        assignments.pop(0)
-
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/client.py:211]: 😭  This changes the Position and Detail History, as it now includes the current assingment.

**Comment by elizabeth-jimenez** [comment]
image

image

**Comment by mjoyce91** [comment]
@elizabeth-jimenez that's correct, but is different with the real web services

**Review by elizabeth-jimenez — APPROVED** [review]
💯🍰🥮🍥🍭  👍

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez that's correct, but is different with the real web services

ooooohhhh, gotcha!

**Pull request merged**